### ISSUE TITLE
Downloaded Bibtext file doesn't always have all citations

### ISSUE THREAD

**Issue by jfy133** [SPAAM-community/AMDirT#67]
Even with with a valid DOI, example: Kocher2021 (single genome) https://doi.org/10.1126/science.abi5658

**Comment by maxibor** [comment]
I'm afraid there is nothing we can do about it: it's a problem with doi.org or science :

For example, for an article of the same issue of science:

```bash
$ curl -L http://doi.org/10.1126/science.abk3132 -H "Accept: application/x-bibtex"
@article{Champenois_2021,
	doi = {10.1126/science.abk3132},
	url = {https://doi.org/10.1126%2Fscience.abk3132},
	year = 2021,
	month = {oct},
	publisher = {American Association for the Advancement of Science ({AAAS})},
	volume = {374},
	number = {6564},
	pages = {178--182},
	author = {E. G. Champenois and D. M. Sanchez and J. Yang and J. P. Figueira Nunes and A. Attar and M. Centurion and R. Forbes and M. Gühr and K. Hegazy and F. Ji and S. K. Saha and Y. Liu and M.-F. Lin and D. Luo and B. Moore and X. Shen and M. R. Ware and X. J. Wang and T. J. Mart{\'{\i}}nez and T. J. A. Wolf},
	title = {Conformer-specific photochemistry imaged in real space and time},
	journal = {Science}
}
```

For `Kocher2021`:

```bash
$ curl -L http://doi.org/10.1126/science.abi5658 -H "Accept: application/x-bibtex"
# nothing
```

**Comment by jfy133** [comment]
Is it

**Comment by maxibor** [comment]
That we can do !

**Comment by alexhbnr** [comment]
Closed by #81 .
